common/changes/@microsoft/gulp-core-build-typescript/ts-4-5_2022-03-17-20-29.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/gulp-core-build-typescript",
+      "comment": "update to typescript 4.5.5",
+      "type": "major"
```

```suggestion
      "comment": "",
      "type": "none"
```

This doesn't actually change anything from a consumer's perspective.